community/blog/2022-04 Interesting new hooks and .NET 6 support in Arcus WebAPI v1.5/README.md

```diff
@@ -0,0 +1,143 @@
+        {
+            var json = JObject.Parse(requestBody);
+            json.Property("clientId").Remove();
+
+            return json.ToString();
+        }
+
+        return requestBody;
+    }
+
+    protected override string SanitizeResponseBody(HttpResponse response, string responseBody)
+    {
+        if (request.HasJsonContentType())
+        {
+            var json = JObject.Parse(responseBody);
+            json.Property("clientName").Remove();
+
+            return json.ToString();
+        }
+
+        return responseBody;
+    }
+}
+```
+
+This custom-made request tracking middleware can then be given to the registration of the request tracking so that next time a request is processed, the personal and/or sensitive information is filtered-out from the tracking.
+
+```csharp
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+
+public class Startup
+{
+    public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
+    {
+        app.UseRouting();
+
+        app.UseRequestTracking<PIIRequestTrackingMiddleware>();
+
+        ...
+        app.UseMvc();
+    }
+}
+```
+
+## Built-in JSON formatting extensions
+Our [Arcus Web API project template](https://templates.arcus-azure.net/features/web-api-template) is using several great and highly recommended approaches to start off your project. One of these approaches is to limit the input/output formatting of models to only JSON formatting. What this means is that the registered input/output formatters were inspected and altered. While this is a valuable addition to the Web API project template, it is still somewhat boilerplate code that with its inspection code doesn't make the template a very user-friendly or clean place to work in.
+
+Arcus Web API v1.5 changed this. In a new library called `Arcus.WebApi.Hosting`, we have added extensions on the MVC options so that this JSON formatting limitation can be added in a more fluent manner.
+
+```csharp
+using System.Text.Json.Serialization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.DependencyInjection;
+
+public void ConfigureServices(IServiceCollection services)
+{
+    services.AddMvc(mvcOptions => 
+    {
+        // Limit the available input/output formatting to only JSON formatting.
+        mvcOptions.OnlyAllowJsonFormatting();
+
+        // Configure that single (JSON) formatting. 
+        mvcOptions.ConfigureJsonFormatting(jsonOptions =>
+        {
+            jsonOptions.IgnoreNullValues = true;
+            jsonOptions.Converters.Add(new JsonStringEnumConverter());
+        }
+    });
+}
+```
+
+These extensions will greatly help make the Arcus Web API project not ony more easier understandable but more maintainable too. If we choose to change internal workings or additional built-in JSON features, we can update our extensions instead of altering the actual internal code of the project template. A cleaner approach for both consumers as maintainers.
```

Comment:
```suggestion
These extensions will greatly help make the Arcus Web API project not only easier to understand but more maintainable too. If we choose to change internal workings or additional built-in JSON features, we can update our extensions instead of altering the actual internal code of the project template. A cleaner approach for both consumers and maintainers.
```